FreeRTOS/FreeRTOS
# Separate FreeRTOS-Cellular-Interface to two parts.

Separate FreeRTOS-Cellular-Interface to two parts.

Description
-----------

1. Update FreeRTOS-Cellular-Interface submodule pointer.
2. Add FreeRTOS-Cellular-Modules submodules' pointers, including BG96 / HL7802 and SARA-R4
3. Update demo project for below updates.

Test Steps
-----------

1. Execute demos in folder:
    - ./FreeRTOS-Plus/Demo/FreeRTOS_Cellular_Interface_Windows_Simulator
2. Fix cellular integration test in folder:
    - ./FreeRTOS-Plus.Test/FreeRTOS-Cellular-Interface

Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
